lesson05/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,359 @@
-html {
-    box-sizing: border-box;
-}
-
 *,
 *:after,
 *:before {
-    box-sizing: inherit;
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+a,
+h2 {
+  font: {
+    family: $font-fam;
+    size: 2rem;
+    weight: 400;
+  }
+  color: $white;
+  text-decoration: none;
+}
+
+body {
+  display: grid;
+  height: 100vh;
+  // grid-template-rows: repeat(28, 1fr);
+  // grid-template-columns: repeat(10, 1fr);
+  grid-template-areas: "header header header header" "header header header header" "nav nav nav nav" "main main main main" "footer footer footer footer";
+}
+
+.header {
+  background-color: black;
+  background-image: url("https://picsum.photos/1200/800");
+  background-size: cover;
+  grid-area: header;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  h1 {
+    color: $white;
+    position: relative;
+    left: 1.5rem;
+    font: {
+      family: $font-fam;
+      size: 6rem;
+      style: italic;
+      weight: 700;
+    }
+    text: {
+      decoration: none;
+      shadow: 2px 2px 5px black;
+    }
+  }
+
+  &__logo {
+    position: relative;
+    left: 5%;
+    color: $white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: teal;
+    border-radius: 50%;
+    width: 7rem;
+    height: 7rem;
+  }
+
+  &__info {
+    position: relative;
+    right: 5%;
+    color: $white;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: black;
+    border-radius: 1rem;
+    border: 1px solid grey;
+    width: 12rem;
+    height: 8rem;
+  }
+}
+
+nav {
+  border: 2px solid $green;
+  background-color: $green;
```

Comment:
Since the border and background color are the same, what benefit is the border providing?

Reply:
Ah, yes. I put that in during development. It should have gotten removed prior to submittal. Thank you!